mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -312,17 +305,22 @@ class BackgroundTasksManager : BroadcastReceiver() {
                         putBoolean(Constants.PREFERENCE_ALARM_CLOCK_LAST_VALUE_WAS_ZERO, time == "0" || time == null)
                     }
 
-                    time
+                    if (time == null) {
+                        null
+                    } else {
+                        ItemUpdateWorker.ValueWithInfo(time, type = ItemUpdateWorker.ValueType.Timestamp)
+                    }
```

Comment:
I know this is my code ;-) but using `time?.let { ItemUpdateWorker.ValueWithInfo(it, type = ...) }` would be more elegant here I think.